### ISSUE TITLE
/cremoveall not working

### ISSUE THREAD

**Issue by PugaBear** [dmulloy2/LWC#18]
Possibly related to #17, /cremoveall does not work. No errors, it says it has removed all protections, but protections are not actually modified. 

I discovered this will attempting to make a /cmodifyall command, the for loop where I get all the protections owned by a player always got 0 protections with the method [lwc.getPhysicalDatabase().loadProtectionsByPlayer(player.getName())](https://github.com/dmulloy2/LWC/blob/master/core/src/main/java/com/griefcraft/sql/PhysDB.java#L1072) which is the same method used by /cremoveall, so I believe the problem lies there.

**Comment by dmulloy2** [comment]
It would appear that this wasn't migrated to UUID, which is a simple fix

**Comment by PugaBear** [comment]
Wow, can't believe that didn't occur to me. Anyways, got that working, I'll create a pull request in minute for /cremoveall, but I've encountered a problem with my /cmodifyall that I can't figure out.

It works, the modifications appear in the database, but it does not show up when doing /cinfo nor does the modification work until the server reboots. I've pretty much copied the [processRightsModification method](https://github.com/dmulloy2/LWC/blob/master/core/src/main/java/com/griefcraft/lwc/LWC.java#L1752-L1868) and removed the sendLocale methods within it. I've also tried adding `protection.saveNow()`, `protection.removeCache()` and `lwc.reloadDatabase()` at the end of the forloop, but no luck. Any idea why its not updating?
